checker/jtreg/nullness/issue824/Class2.java

```diff
@@ -1,9 +1,10 @@
 /*
  * @test
  * @summary Test case for Issue 824 https://github.com/typetools/checker-framework/issues/824
+ * The defaults for type variable upper bounds with type Object changed since the issue was filed.  So, this test case has been changed so that annotations on type variable bounds in stub files is still tested.
```

Comment:
In the new GitHub code review process, I'm not sure how to mark this as resolved so it no longer shows up in the conversation.  Do you know?